WalletWasabi.Backend/wwwroot/about.html

```diff
@@ -0,0 +1,281 @@
+    <meta property="twitter:image" content="https://wasabiwallet.io/img/wasabi-social.jpg">
+
+    <link rel="stylesheet" href="css/bootstrap.css">
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="shortcut icon" href="img/favicon.ico">
+    <title>Wasabi Wallet - Bitcoin privacy wallet with built-in coinjoin</title>
+</head>
+
+<body class="fixed-heading">
+
+    <nav class="navbar navbar-expand-lg navbar-dark">
+        <div class="container">
+            <a class="logo" href="index.html"><img src="img/logo.svg" alt="Wasabi Wallet">
+            </a>
+            <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+                <span class="navbar-toggler-icon"></span>
+            </button>
+            <div class="collapse navbar-collapse" id="navbarSupportedContent">
+                <ul class="navbar-nav ms-auto mb-2 mb-lg-0">
+                    <li class="nav-item">
+                        <li class="nav-item">
+                            <a class="nav-link" href="index.html">The Wallet</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="contribution.html">Contribution</a>
+                        </li>
+                        <li class="nav-item active">
+                            <a class="nav-link" href="about.html">About us</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="https://blog.wasabiwallet.io" target="_blank">Blog</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="faq.html">Help</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="btn btn-secondary" href="index.html#download">Download</a>
+                        </li>
+                </ul>
+            </div>
+        </div>
+    </nav>
+
+    <div class="fixed-img">
+        <img src="img/05_wallet.png" alt="Wasabi Wallet - About">
+        <div class="floating-title">Better money.
+            <br>Better world.</div>
+    </div>
+
+    <div class="content">
+        <section class="blured-section p-100">
+            <div class="container">
+                <div class="row">
+                    <div class="col-md-8">
+                        <h1 class="title-lined left">Motivation</h1>
+                        <p>
+                            We believe that the most urgent problem for mankind is fixing the money, that Bitcoin is the best option for doing this and that privacy is its biggest problem.
```

Comment:
```suggestion
                            We believe that the world is sick because the money is poisoned, that Bitcoin is the best option for fixing the money and that privacy is its biggest problem.
```

removes twice `problem`